composer.json

```diff
@@ -11,15 +11,15 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/lock": "^3.4|^4.0",
-        "symfony/cache": "^3.4|^4.0",
-        "symfony/http-foundation": "^3.0|^4.0",
-        "symfony/http-kernel": "^3.0|^4.0",
-        "symfony/options-resolver": "^3.0|^4.0"
+        "symfony/lock": "^3.4|^4.0|^5.0",
+        "symfony/cache": "^3.4|^4.0|^5.0",
+        "symfony/http-foundation": "^3.0|^4.0|^5.0",
+        "symfony/http-kernel": "^3.0|^4.0|^5.0",
+        "symfony/options-resolver": "^3.0|^4.0|^5.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.12",
-        "phpunit/phpunit": "^5.7.27",
+        "symfony/phpunit-bridge": "dev-master",
```

Comment:
Is there no stable release yet that works for us?

Reply:
The `SetUpTearDownTrait` is needed to support this wide range on php versions with phpunit. This was  added https://github.com/symfony/symfony/pull/32922 which is only available in 4.4.

I can also workaround this problem by refractoring the tests that no `tearDown` and `setUp` is needed.

Reply:
Ok good fix it by set the `SYMFONY_PHPUNIT_VERSION` see https://github.com/Toflar/psr6-symfony-http-cache-store/pull/18/files#diff-180cc0a648d9fd0a0ca3ede96ea0aeef